src/ContainerLogs.jsx

```diff
@@ -165,8 +166,21 @@ class ContainerLogs extends React.Component {
 
     render() {
         let element = <div className="container-logs" ref={this.logRef} />;
-        if (this.state.errorMessage)
+        const { systemd_unit, uid } = this.props;
+
+        if (this.state.errorMessage) {
             element = <EmptyStatePanel icon={ExclamationCircleIcon} title={this.state.errorMessage} />;
+        } else if (systemd_unit !== undefined && uid === 0) {
```

Comment:
Not sure if it matters with JS but `uid===0` seems like it would be faster as it's a number comparison vs `systemd_unit` being a string comparison, so changing the order of those:
```suggestion
        } else if (uid === 0 && systemd_unit !== undefined) {
```

And can `systemd_unit` be empty string and be acceptable? Otherwise I'd shorten the suggestion even more 
```suggestion
        } else if (uid === 0 && systemd_unit) {
```